graveler/graveler.go

```diff
@@ -321,6 +334,15 @@ type VersionController interface {
 	SetHooksHandler(handler HooksHandler)
 }
 
+// Plumbing includes commands for fiddling more directly with graveler implementation
+// internals.
+type Plumbing interface {
+	// GetMetarange returns information where metarangeID is stored.
+	GetMetaRange(ctx context.Context, repositoryID RepositoryID, metaRangeID MetaRangeID) (MetaRangeData, error)
+	// GetRange returns information where rangeID is stored.
```

Comment:
Consider calling it xxxStats returning xxxInfo
ex: MetaRangeStat -> MetaRangeInfo